src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -170,4 +189,19 @@ public void equals() {
         assertFalse(standardCommand.equals(new EditCommand(INDEX_FIRST_PERSON, DESC_BEN)));
     }
 
+    @Test
+    public void equals_sameObject_true() {
+        assertEquals(EDIT_PERSON_DESCRIPTOR, EDIT_PERSON_DESCRIPTOR);
```

Comment:
May I know what is the point of this test actually? It has the same name pointing to the same object. It will always result in true.

Reply:
My guiding principle is doing the bare minimum to meet Jacoco requirements. So if IntelliJ says this == this needs to be covered, then I cover it.

Reply:
@russelltankaimin If you check the `equals()` method in `EditPersonDescriptor`, it is overridden to check all the values in it.